From ebe63a9c78a2db294e971df681bf8da3d74588b8 Mon Sep 17 00:00:00 2001
Message-ID: <ebe63a9c78a2db294e971df681bf8da3d74588b8.1732384228.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732384228.git.sharlatanus@gmail.com>
References: <cover.1732384228.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 14 Nov 2024 22:08:20 +0000
Subject: [PATCH 21/46] gnu: python-ginga: Update to 5.2.0.

* gnu/packages/astronomy.scm (python-ginga): Update to 5.2.0.
[propagated-inputs]: Add python-puremagic.

Change-Id: Ibdb5e3057365ddd67efa49f0ace39912b330be71
---
 gnu/packages/astronomy.scm | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index c10fdc14f6..21adcd42a0 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2238,13 +2238,13 @@ (define-public python-gatspy
 (define-public python-ginga
   (package
     (name "python-ginga")
-    (version "5.1.0")
+    (version "5.2.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "ginga" version))
        (sha256
-        (base32 "0c6vpcz59x889z8ggq6r5ci9cf10m7r9h262r6chx31mzdr010wf"))))
+        (base32 "0gdb4wkz3ivizyb15x86q3z9ms7mlhpakwksp417mgy1av85wssh"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -2276,6 +2276,7 @@ (define-public python-ginga
            python-packaging
            python-photutils
            python-pillow
+           python-puremagic
            python-pyyaml
            python-qtpy
            python-scipy
-- 
2.46.0

